headers/functions/overlay29.h

```diff
@@ -366,9 +375,20 @@ bool TryInflictLeechSeedStatus(struct entity* user, struct entity* target, bool
                                bool check_only);
 void TryInflictDestinyBond(struct entity* user, struct entity* target);
 void TryInvisify(struct entity* user, struct entity* target);
+void TryIncreaseBelly(struct entity* user, struct entity* target, int belly_restoration,
+                      int max_belly_boost, bool log_failure);
 void TryTransform(struct entity* user, struct entity* target);
+bool TryInflictBlinkerStatus(struct entity* user, struct entity* target, bool check_only,
+                             bool log_failure);
 bool IsBlinded(struct entity* entity, bool check_held_item);
+bool TryInflictCrossEyedStatus(struct entity* user, struct entity* target, bool check_only);
+void TryInflictEyedropStatus(struct entity* user, struct entity* target);
+bool TryInflictSlipStatus(struct entity* user, struct entity* target);
 void RestoreMovePP(struct entity* user, struct entity* target, int pp, bool suppress_logs);
+void ApplyProteinEffect(struct entity* attacker, struct entity* defender, int stat_boost);
```

Comment:
Does it actually make sense to call these `attacker` and `defender`? I would think all these item effects should be `user` and `target` instead